app/src/main/res/values/strings.xml

```diff
@@ -289,6 +289,7 @@
     <string name="MUSIC_TURN_REPEAT_ALL_ON">Turn Repeat All On</string>
     <string name="MUSIC_TURN_REPEAT_ONE_ON">Turn Repeat One On</string>
     <string name="MUSIC_TURN_REPEAT_OFF">Turn Repeat Off</string>
+    <string name="MUSIC_TEMPORARY_PLAYLIST_DESCRIPTION">AndroidAutoIDrive dummy playlist - please do not delete</string>
```

Comment:
I'd prefer a wording like "AndroidAutoIDrive temporary playlist", and I'm not sure the "please do not delete" is strictly necessary: To me, it gives the impression that Bad Things Will Happen if it is deleted, as opposed to a slight de-optimization path. I think the effort to recreate it is only slightly more than the effort to replace the contents with any updated Liked Songs contents.

Reply:
Updated the wording on that and removed the "please do not delete" part. You're right, it isn't really necessary and if the user does delete it recreation really isn't that intense of an operation. I haven't measured the difference in the time it takes to replace the contents of a temporary playlist in comparison to the full creation of a playlist but I think you're right that they would be very similar. I should eventually run some performance metrics and see how all of the possible flows compare, mostly out of curiosity 😃